plugins/scaffolder-backend-module-github/api-report.md

```diff
@@ -128,6 +128,7 @@ export interface CreateGithubPullRequestActionOptions {
       } | null>;
     }
   >;
+  config: Config;
```

Comment:
I think that we want to make this optional otherwise it's a breaking change. Should be fine to be able to support optional config in the object too.

Reply:
done: https://github.com/backstage/backstage/pull/24628/commits/18cd46471f2936027cd2492523892f27bbe4bf3b